dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ConsumerModel.java

```diff
@@ -64,7 +66,20 @@ public ConsumerModel(String serviceKey
         this.referenceConfig = referenceConfig;
     }
 
-    public void init(Map<String, AsyncMethodInfo> attributes) {
+
+    public void initMethodModels(List<MethodConfig> methodConfigs) {
+
+        Map<String, AsyncMethodInfo> attributes = null;
+        if (CollectionUtils.isNotEmpty(methodConfigs)) {
+            attributes = new HashMap<>(16);
+            for (MethodConfig methodConfig : methodConfigs) {
+                AsyncMethodInfo asyncMethodInfo = methodConfig.convertMethodConfig2AsyncInfo();
```

Comment:
I think the model class should be as simple as possible, it is best to convert `MethodConfig` first, and then pass it to `ConsumerModel`.

Reply:
In fact, ConsumerModel is not that simple, and the initMethodModels method of ConsumerModel is for MethodModel. There is nothing wrong with the association between MethodModel and MethodConfig.

Reply:
There are still some problems:
* `ConsumerMethodModel` and `AsyncMethodInfo` are not the same things, the method naming is not very appropriate `initMethodModels(List<MethodConfig> methodConfigs)`, the origin method name is `init()`, it's means do some initialization, may include other attributes later. 
* `List<MethodConfig> methodConfigs` can be obtained by `ReferenceConfig.getMethods()`. 
In fact, it does not need to be passed in by parameters.

Reply:
Agree with this point of view, I revised it and reviewed it again.